include/algo_helpers.h

```diff
@@ -126,6 +127,8 @@ template<class Vector>
 float min_seg_dist(const Vector &B0, const Vector &B0p, const Vector &B1,
                    const Vector &B1p,float &s,float &t);
 
+void write_STL(ostream &file, const Tube<Vector3> &tube, const int S);
```

Comment:
My very vague recollection of this file that it contained helper things to run simulations, not really I/O aspects of curves/tubes.

Apparently most of the read*, write* methods are in Tube/Curve directly. Just move writeSTL into Tube?

Reply:
Done.